### DISCUSSION TITLE
GET_FLAT_FILE_ACTIONABLE_ORDER_DATA_SHIPPING Report

### DISCUSSION THREAD

**Discussion by insoledept** [abuzuhri/Amazon-SP-API-CSharp#139] [Q&A]
First off, congratulations on a fantastic project.  

I am curious if someone could provide an example where the RDT token is used to download the Unshipped Orders report in the case of PII?  This request is very similar to another on the board, but wanted to chime in since we could use it too. I have tried with the standard LWA token, but of course, this did not work, as this report contains PII. However, other reports complete with success.  

Many thanks in advance!

**Comment by feetpeoplellc** [comment]
Is it possible to download a report with PII at this stage?
